plugins/techdocs-node/src/stages/publish/azureBlobStorage.ts

```diff
@@ -78,6 +78,26 @@ export class AzureBlobStoragePublish implements PublisherBase {
       );
     }
 
+    const legacyPathCasing =
+      config.getOptionalBoolean(
+        'techdocs.legacyUseCaseSensitiveTripletPaths',
+      ) || false;
+
+    // Give more priority for azurite, if configured, return the AzureBlobStoragePublish object here itself
```

Comment:
Calling it `azuriteConnectionString` misleadingly implies a hard dependency on Azurite. I suggest calling it `connectionString`.